core/base/ftmTree/FTMTreeUtils_Template.h

```diff
@@ -10,44 +10,35 @@
 
 #pragma once
 
-//#include <FTMTreeMT.h>
-
 namespace ttk {
   namespace ftm {
     // --------------------
-    // Get
+    // Utils
     // --------------------
+    template <typename type>
+    static type myAbs(const type var) {
+      return (var >= 0) ? var : -var;
```

Comment:
hum. in principle, ``std::abs`` should work for most of the data types supported by vtk (and hence ttk).
so that function could be removed.

Reply:
`std::abs` throw an error if we try to apply this to a template variable since it can't be apply to unsigned int. However we can still cast the variable before using abs. It will done in the next commit.